src/Java.Interop/Java.Interop/JavaException.cs

```diff
@@ -249,12 +236,12 @@ void IJavaPeerable.Finalized ()
 
 		void IJavaPeerable.SetJniIdentityHashCode (int value)
```

Comment:
@jonpryor Also wondering whether `IJavaPeerable` needs set methods  like `SetJniIdentityHashCode` or whether we might turn them into property setters?